README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
No, it's really `.bower.json` :) `.bower.json` is the metadata file added by Bower when a bower component is installed. Therefore, it does not exist on installed npm packages. This is also the suggested descriptor as seen [here](https://webpack.github.io/docs/usage-with-bower.html).

Some (isomorphic) npm packages do not npmignore `bower.json` and it's included in the installed package. Thus, webpack finds the node_module, but uses the bower.json file descriptor. Same applies for bower components that include the package.json. For this, it cannot be:

```
modules: ['bower_components', 'node_modules'],
descriptionFiles: ['package.json', 'bower.json']
// will use bower_components/**/package.json if available
```

or

```
modules: ['node_modules', 'bower_components'],
descriptionFiles: ['bower.json', 'package.json']
// will use node_modules/**/bower.json if available
```